CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -195,6 +199,22 @@ static function X2MissionSourceTemplate GetCovertMissionSource(X2CovertMissionIn
 	return X2MissionSourceTemplate(StratMgr.FindStrategyElementTemplate(MissionInfo.MissionSource));
 }
 
+static function StrategyCost GetExfiltrationCost(XComGameState_CovertAction CovertAction)
+{
+	local StrategyCost ExfiltrateCost;
+	local ArtifactCost IntelCost;
+	local TDateTime CurrentTime;
+
+	CurrentTime = class'XComGameState_GeoscapeEntity'.static.GetCurrentTime();
+
+	IntelCost.Quantity = default.EXFIL_COST_BASEAMOUNT + (class'X2StrategyGameRulesetDataStructures'.static.DifferenceInHours(CurrentTime, CovertAction.StartDateTime) / 24) * default.EXFIL_COST_MULTIPLIER;
```

Comment:
1) you need to save days to a `float` variable
2) multiply it by `EXFIL_COST_MULTIPLIER`
3) round the result, add to base